nengo_gui/components/pointer.py

```diff
@@ -76,6 +80,7 @@ def javascript(self):
     def code_python_args(self, uids):
         return [uids[self.obj], 'target=%r' % self.target]
 
+    # Override the value if set
```

Comment:
This comment doesn't make sense to me at this place. Which value is overridden? Why does `message` override a value instead of sending a message?